src/Select/index.tsx

```diff
@@ -1,10 +1,14 @@
 import { Select as AntdSelect, SelectProps as AntdSelectProps } from 'antd';
+import { BaseOptionType, DefaultOptionType } from 'antd/lib/select';
 import React, { ReactElement, useCallback } from 'react';
 import styled from 'styled-components';
 
 import { fontSizeFromTheme } from '../styled-utils';
 
-export type SelectProps<T> = AntdSelectProps<T>;
+export type SelectProps<
+  ValueType = unknown,
```

Comment:
Should it have a default?
```suggestion
  ValueType,
```

Reply:
I think so, it is more convenient that way. Sometimes the value is not important.